src/game_battle.h

```diff
@@ -189,6 +192,13 @@ namespace Game_Battle {
 
 	/** @return death teleport handler if one is installed, otherwise an inactive target */
 	TeleportTarget GetDeathHandlerTeleport();
+
+	/** Don't reference this, use IsBattleRunning()! */
+	extern bool battle_running;
```

Comment:
Can't you move this in an anonymous namespace to prevent the access?

Reply:
Unfortunately not because it's in the header. I share your concerns, but I wanted to make sure access gets inlined.

This could be changed from namespace to struct and then it made private. But I did not want to make such a big change at this time.